src/qiboconnection/typings/responses/job_response.py

```diff
@@ -1,26 +0,0 @@
-""" JobResponse """
-from dataclasses import dataclass
-
-from ..enums import JobStatus
-from ..requests import JobRequest
-
-
-@dataclass
-class JobResponse(JobRequest):
-    """Full Job Response. Includes job results which may
-    be weight a few GB.
-
```

Comment:
if this was moved only bc of the from kwargs constructor, it can aswell stay with the rest of resposnes.
For me it makes more sense that it stays within the classes that directly represent responses. Despite it having _some_ logic, we are not operating with it. Only type-casting jsons.
But this is arguable, what do you think?

Reply:
this makes sense to mey, yes? Moved it back to responses